### DISCUSSION TITLE
Is ot possible to make parts of platform open for internet

### DISCUSSION THREAD

**Discussion by Arkronus** [exoplatform/exo-community#61] [Q&A]
Hi!

Is it possible to make spaces on the platform visible for all internet users without registration? This way, it would be easier to grow my community and ease sharing of content for my users.

**Comment by rdenarie** [comment]
Hi @Arkronus 

Since eXo Platform 6.5, it is possible to have a public site, accessible to all users.
But, to be able to join a space, a registration is mandatory. 
As the space provide a lot a features driven by permissions, we have to know who is the user when he join a space. So it is not currently possible to have space publicly opened.
